projects/ad9361/src/main.c

```diff
@@ -503,6 +503,9 @@ int main(void)
 	}
 	gpio_direction_output(default_init_param.gpio_desc_resetb, 1);
 
+	rx_adc_init.base = AD9361_RX_1_BASEADDR;
+	tx_dac_init.base = AD9361_TX_1_BASEADDR;
```

Comment:
just to be sure: this won't break other implementations? [i.e. non-fmcomms5 implementations]

Reply:
This is included in the compiler switch for FMCOMMS5.

Reply:
A workaround would be to add separate functions / if condition in the AD9361_init function that set these parameters based on current `id_no` (similar to the older version).

Reply:
nevermind;
just saw the FMCOMMS5 #ifdef